packages/base-map/src/index.story.js

```diff
@@ -183,5 +183,9 @@ export const customLocationPopupContent = () => (
     onPopupClosed={onPopupClosed}
   />
 );
+// Custom styling for this story only, not in production
+customLocationPopupContent.parameters = {
+  a11y: { config: { rules: [{ id: "color-contrast", reviewOnFail: true }] } }
+};
```

Comment:
Reusing a11y rules defined in other packages is tricky on this one...

Reply:
Have extracted it into a const for consistency in 59cfcfb631e5a0f6c6ef52bc7260faf988bcd5bd